### DISCUSSION TITLE
Requirements for Feature Mortgage Calculator

### DISCUSSION THREAD

**Discussion by acantley** [adesokan01/T24S2T1EXTG01#4] [General]
I am progressing through the development of the Mortgage Calculator and would like to clarify that I am correctly understanding the requirements.

Am I correct in thinking that the **'Monthly Interest Rate'** input text box is only to display output and not allow the user to input the monthly interest rate? (i.e. the Monthly Interest Rate is calculated from the Yearly Interest Rate on click of the Calculate button).

**Comment by adesokan01** [comment]
Yes, you are correct. The 'Monthly Interest Rate' text box in the mortgage calculator application is used to display the calculated value rather than accept user input. 
The 'Monthly Interest Rate' text box is meant to show the result of a calculation rather than receive input. Specifically, it displays the monthly interest rate that is derived from the yearly interest rate.
